src/index.html

```diff
@@ -10,20 +10,18 @@
 		<!--- TODO : <meta name="description" content="" />-->
 
 		<!-- Facebook Meta Tags -->
-		<!-- TODO: <meta property="og:url" content="" />-->
+		<meta property="og:title" content="Antiplag">
+		<meta property="og:description" content="Check your files before submitting.">
```

Comment:
I'm not sure, whether this should be in slovak or english.

Reply:
I think English is OK, or should be both slovak and englis?

Reply:
Not sure, I think it's an question for our PM , you better ask him.